src/olympia/versions/tests/test_admin.py

```diff
@@ -5,47 +5,32 @@
 
 from pyquery import PyQuery as pq
 
-from olympia.amo.tests import TestCase, addon_factory, user_factory
+from olympia import amo
+from olympia.amo.tests import TestCase, addon_factory, user_factory, version_factory
 from olympia.reviewers.models import NeedsHumanReview
-from olympia.versions.models import InstallOrigin
+from olympia.versions.models import InstallOrigin, VersionProvenance
 
 
 class TestVersionAdmin(TestCase):
-    def test_authorized_user_has_access(self):
-        addon = addon_factory()
-        version = addon.current_version
-        detail_url = reverse('admin:versions_version_change', args=(version.pk,))
-        user = user_factory(email='someone@mozilla.com')
-        self.grant_permission(user, 'Admin:Advanced')
-        self.client.force_login(user)
-        response = self.client.get(detail_url, follow=True)
-        assert response.status_code == 200
-
-        doc = pq(response.content)
-        assert doc('textarea#id_release_notes_0').length == 1
-
-    def test_unauthorized_user_has_no_access(self):
-        addon = addon_factory()
-        version = addon.current_version
-        detail_url = reverse('admin:versions_version_change', args=(version.pk,))
-        user = user_factory(email='someone@mozilla.com')
-        self.client.force_login(user)
-        response = self.client.get(detail_url, follow=True)
-        assert response.status_code == 403
+    def setUp(self):
```

Comment:
NIT: more readable to split the refactoring here to a separate commit so one can distinguish the refactor from the addition. No change needed just mentioning.